app/views/participants/index.js.erb

```diff
@@ -1,7 +1,10 @@
 $("#participants_table")
-  .html("<%=j render('participants_table') %>")
   .removeClass("loading")
   .attr("data-report", "<%= @report %>")
+  .addClass("<%= @report %>")
```

Comment:
As far as I can tell, this is just set in the controller to whatever the parameter is, and might be an XSS vector